tests/deploy_test.go

```diff
@@ -121,6 +123,30 @@ func Test_Deploy_WithAnnotations(t *testing.T) {
 	}
 }
 
+func Test_DeploymentNamespaces(t *testing.T) {
+	defaultNamespace := "openfaas-fn"
+	expectedNamespaces := config.Namespaces
+	expectedNamespaces = append(expectedNamespaces, defaultNamespace)
+	actualNamespaces, err := config.Client.ListNamespaces(context.Background())
+
+	if err != nil {
+		t.Fatalf("Unablen to List Openfaas Namespaces %s", err)
+	}
+
+	if len(expectedNamespaces) != len(actualNamespaces) {
+		t.Fatalf("Expected %v but got %v", expectedNamespaces, actualNamespaces)
+	}
+
+	sort.Strings(expectedNamespaces)
+	sort.Strings(actualNamespaces)
+
+	for i, ns := range expectedNamespaces {
+		if ns != actualNamespaces[i] {
+			t.Fatalf("Expected %v but got %v", expectedNamespaces, actualNamespaces)
```

Comment:
Similarly, let's make the message a little more specific 
```suggestion
			t.Fatalf("want namespace: %q , got %q", expectedNamespaces, actualNamespaces)
```